bcwallet/bcwallet.py

```diff
@@ -1269,8 +1275,11 @@ def cli():
             puts(colored.red("Invalid wallet entry: %s" % wallet))
 
     else:
-        puts("You've opened your HD wallet without specifying a master public or master private key, which you can do like this:\n")
-        print_bcwallet_basic_priv_opening(priv_to_display='xpriv123...')
+        puts("You've opened your HD wallet without specifying a master public or master private key.\n")
+        puts("To open your wallet with your master private key:\n")
+        print_bcwallet_basic_priv_opening(priv_to_display='xpriv123...\n')
+        puts("To open your wallet with your master public key:\n")
+        print_bcwallet_basic_priv_opening(priv_to_display='xpub123...\n')
```

Comment:
You don't think that's TMI? I'm especially hesitant to push public (watching only) mode because it doesn't yet support airgapped transaction signing (though it's easy enough to add).